qiita_pet/support_files/doc/source/processingdata/woltka_pairedend.rst

```diff
@@ -0,0 +1,62 @@
+Wolka and Bowtie2 using Read Pairing Schemes
+============================================
+
+Benchmarks created by Qiyun Zhu (@qiyunzhu) on Aug 1, 2024.
+
+Summary
+-------
+
+I tested alternative read pairing schemes in the analysis of shotgun metagenomic sequencing data. Sequencing reads were aligned against a reference microbial genome database as unpaired or paired, with or without singleton and/or discordant alignments suppressed. A series of synthetic datasets were used in the analysis.
```

Comment:
"singleton and/or discordant alignments" is now irrelevant. It needs to be removed from this paragraph and the next two.